app/views/admin/sitewide_settings/_form.html.erb

```diff
@@ -1,11 +1,45 @@
-<p class="warning">Warning: changes to sitewide settings appear instantly on the live site.</p>
-<p><%= govspeak_to_html sitewide_setting.description%></p>
+<%= render "govuk_publishing_components/components/warning_text", {
+  text: "Changes to sitewide settings appear instantly on the live site.",
+  highlight_text: true
+} %>
 <%= form_for [:admin, sitewide_setting], as: :sitewide_setting do |form| %>
-  <%= form.errors %>
-  <fieldset>
-    <p><%= form.check_box :on%></p>
-    <p><%= form.text_area :govspeak, rows: "8"%></p>
-    <p>Please check this is valid govspeak. </p>
-  </fieldset>
-  <%= form.save_or_cancel(cancel: admin_sitewide_settings_path) %>
-<% end %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <p class="govuk-body"><%= govspeak_to_html sitewide_setting.description%></p>
+      <%= render "govuk_publishing_components/components/radio", {
+    heading: "Enable re-shuffle mode?",
+    heading_size: "l",
```

Comment:
and this and the component below needs the indentation fixed. As i said on slack i'll ask Ollie about potentially adding erb lint and see what he thinks.

Reply:
```
     <%= render "govuk_publishing_components/components/radio", {
        heading: "Enable re-shuffle mode?",
        heading_size: "l",
        name: "sitewide_setting[on]",
        id: "sitewide_setting_on",
        items: [
          {
            value: "1",
            checked: sitewide_setting.on,
            text: "Yes"
          },
          {
            value: "0",
            checked: !sitewide_setting.on,
            text: "No"
          }
        ]
      } %>

      <%= render "govuk_publishing_components/components/textarea",
        label: {
          heading_size: "l",
          text: "Can you provide more detail in Govspeak?"
        },
        name: "sitewide_setting[govspeak]",
        id: "sitewide_setting_govspeak",
        error_message: errors_for_input(sitewide_setting.errors, :govspeak),
        hint: "Please check this is valid govspeak.",
        value: sitewide_setting.govspeak
        %>

      <div class="govuk-button-group">
        <%= render "govuk_publishing_components/components/button", {
          text: "Save"
        } %>

        <%= link_to("cancel", admin_sitewide_settings_path(), class: "govuk-link") %>
      </div>
    </div>
```